packages/xo-web/src/xo-app/hub/resource.js

```diff
@@ -75,6 +76,12 @@ export default decorate([
           size: 'medium',
         })
 
+        resourceParams.pools.forEach(pool => {
```

Comment:
You can do it here https://github.com/vatesfr/xen-orchestra/pull/4571/files#diff-17032e40a62de5b80aa40071c01441faR93